.github/workflows/ruby.yml

```diff
@@ -12,9 +12,18 @@ jobs:
     - name: Set up Ruby 2.7
       uses: actions/setup-ruby@v1
       with:
-        ruby-version: 2.7.1
-    - name: RuboCop
+        ruby-version: 2.7
+    - name: Cache gems
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-rubocop-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-rubocop-
+    - name: Install gems
       run: |
-        gem install bundler
+        bundle config path vendor/bundle
+        bundle config set without 'pages'
         bundle install --jobs 4 --retry 3
-        bundle exec rubocop
+    - name: RuboCop
+      run: bundle exec rubocop --parallel
```

Comment:
Why `--parallel`?

Reply:
It's faster. E.g. look at [Rails](https://github.com/rails/rails/blob/100f7a7f643ae6766f406ccf1e049050199caddc/.github/workflows/rubocop.yml#L28) (again) and [Rubocop](https://github.com/rubocop-hq/rubocop/blob/v0.84.0/Rakefile#L21-L27).  

What is your concern?

Reply:
I understand the optimization but we only have a couple of `.rb` in this repo, and they are very small. We don't need to run cops in parallel, it's not gonna give us any significant boost in the check speed.

Reply:
It's just a generally faster approach. Of course, we don't care about `time rubocop` in this repo at the moment. OK, changed.